react/src/views/BoxEdit/BoxEditView.tsx

```diff
@@ -141,11 +142,36 @@ function BoxEditView() {
       },
     })
       .then((mutationResult) => {
-        navigate(`/bases/${baseId}/boxes/${mutationResult.data?.updateBox?.labelIdentifier}`);
+        if (mutationResult?.errors) {
+          notificationVar({
+            title: `Box ${labelIdentifier}`,
+            type: "error",
+            message: "Error while trying to update Box",
+          });
+        } else {
+          notificationVar({
+            title: `Box ${labelIdentifier}`,
+            type: "success",
+            message: `Successfully modified with ${
```

Comment:
@aerinsol success message after box update: e.g. Successfully modified with baby towels Unisex Baby (75x) in LOST